2repository/build.gradle

```diff
@@ -38,48 +39,33 @@ subprojects {
     def parentImage = parentProject.getTasks().getByName('buildDockerImage')
 
     dockerAlfresco {
-        baseImage = { parentImage.getImageId().get() }
+        baseImage = parentImage.getImageId()
         dockerBuild {
-            if (!(project.hasProperty("repoName"))) {
-                ext.repoName = "hub.xenit.eu"
-            }
-            def namespace = (project.alfresco.flavor == "enterprise" ? "alfresco-enterprise" : "public");
-            repository = ((repoName == "hub.xenit.eu") ?
-                    "${repoName}" + "/" + namespace + "/" + "alfresco-repository-${project.alfresco.flavor}" :
-                    "${repoName}" + "/" + "alfresco-repository-${project.alfresco.flavor}")
+            repository = calcRepository(project.alfresco.flavor)
             tags = calcTags(project.alfresco.version)
             pull = false
-            automaticTags = false
         }
     }
 
-    afterEvaluate {
-        createDockerFile.dependsOn(parentImage)
-
-        if (project.hasProperty('testsChangedPorts') && project.testsChangedPorts)
-            changedPortsComposeUp.dependsOn(buildDockerImage)
-        else
-            changedPortsComposeUp.enabled = false
+    createDockerFile {
+        smartCopy(configurations.sharedLib, '${CATALINA_HOME}/shared/lib/')
+        if (!configurations.sharedBin.isEmpty())
+            smartCopy(tarTree(resources.gzip(configurations.sharedBin.singleFile)), '/opt/alfresco')
```

Comment:
sharedBin is always a single file ?

Reply:
It was already like that before. It is a `.tar.gz` file that needs to be unzipped before being extracted.
It is the `alfresco-pdf-renderer`, so perhaps I can rename the configuration.

https://github.com/xenit-eu/docker-alfresco/blob/9a03886f2cf50b7fc9f2c2c2505f891179f1338c/3legacy/5.2/enterprise-5.2.5/overload.gradle#L14